src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Utility.cs

```diff
@@ -29,8 +29,8 @@ public static string EncryptUsingGatwewaySettings(GatewayStatus status, string s
             var result = string.Empty;
             var jwk = status.EncryptionJwk;
 
-#if PORTABLE
-            using (var rsa = RSA.Create())
+#if NET45
```

Comment:
Can we make something like below, this way we do not lose the existing Else part code.
#if NET45
#elseIf Portable
#else